packages/commonwealth/server/routing/router.ts

```diff
@@ -145,6 +145,7 @@ import { ServerReactionsController } from '../controllers/server_reactions_contr
 import { ServerThreadsController } from '../controllers/server_threads_controller';
 import { ServerTopicsController } from '../controllers/server_topics_controller';
 
+import { TBC_BALANCE_TTL_SECONDS } from 'server/config';
```

Comment:
This import is breaking CI, `yarn start`, and may also break deployments.
```suggestion
import { TBC_BALANCE_TTL_SECONDS } from '../config';
```